src/smc-webapp/project_page.cjsx

```diff
@@ -507,7 +507,7 @@ exports.ProjectPage = ProjectPage = rclass ({name}) ->
                     distance             = {3 if not IS_MOBILE}
                     bsStyle              = "pills"
                     style                = {display:'flex', overflow:'hidden', flex: 1}
-                    shouldCancelStart    = {(e)=>e.target.getAttribute('class')?.includes('smc-file-tabs')}
+                    shouldCancelStart    = {(e)=>'smc-file-tabs' in e.target.getAttribute('class')?}
```

Comment:
This is seriously wrong, e.g.,

```
coffee> 0 in [0]?
false
```

Unfortunately, you have to do something more complicated with an if...